README.md

```diff
@@ -30,22 +27,45 @@ Use the `-port` option to specify the port number for the server to listen reque
 ./code-runner-server -port <port>
 ```
 
-## Send code files and arguments to server
-Use curl command to connect to the server and send request.
-Use multipart/form data option(-F) to provide the assignment submission file and the command-line arguments that are to be used when executing the code.
+## Run Code Runner inside a Docker Container
+- Prerequisite for this is that docker engine should be installed.
+- See [instructions](https://docs.docker.com/engine/installation/) for installing docker engine on different supported platforms.
+- A docker image of code runner is used as a base image for different assignment environment images.
+- Code Runner base image is not directly used, instead the image pertaining to the required assignment environment should be used to run the assignments.
+
+Use the following docker command to run the required assignment environment image.
 ```commandline
-curl <server_ip_address:port_number> -F <compressed_filepath> -F <arg1> -F <arg2> ...
+docker run --publish <port_to_expose>:<port_to_run> <image_tag> -port <port_to_run>
 ```
 
+## Supported Languages
+Below is the list of supported languages and the corresponding versions.
+- gcc 7
+- g++ 7
+- python 3.7
+- java 8 & 11
+
 ## Web page to Submit Assignment
-![](docs/codeRunnerPage.png)
+Use the exposed port number to open the web page. Example - for exposed port number as 52453, hit `http://localhost:52453` on browser.
```

Comment:
Maybe make the url a link? This will help them click on it and see the application landing page.
`[http://localhost:52453]()`

Reply:
Sure.

Reply:
Okay. But unless the container is running, `localhost:52453` won't display anything. So maybe I can open index.html page on click of the link as `[http://localhost:52453](client/index.html)`

Reply:
Maybe displaying the index.html on click of the link can be done - `[http://localhost:52453](https://github.com/assignment-exec/code-runner/blob/master/client/index.html)`